tests/system_test_list.txt

```diff
@@ -74,3 +74,8 @@ standard/plate_hole.json
 standard/navier_stokes_static.json
 standard/navier_stokes_transient.json
 standard/stokes_static.json
+standard/neohookean.json
+standard/mooney_rivlin.json
+#standard/mooney_rivlin_p2.json
```

Comment:
This one is not working with LDLT linear solver but works with the BiCGStab solver. I leave it as future debugging.